styles/main.css

```diff
@@ -418,8 +419,11 @@ div.gallery a.col-span-2{
     grid-column: span 2;
 }
 
+div.gallery a {
+    transition: transform 200ms;
+}
 div.gallery a:hover{
-    scale: 1.1;
+    transform: scale(1.1);
```

Comment:
`scale` only works in Firefox.  Best to stick with the widely supported `transform` property